vogue/commands/load/application_tag.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import click
+import json
+from vogue.load.application_tag import load_aplication_tags
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("apptag", short_help="Reads json with application tag info.")
+@click.option('-a', '--application-tags', required=True, help='json formatted application tags')
```

Comment:
I don't understand why this is an option. It's the only required option, so why not use an argument.

If it is a file, you can use click.File to already open the file.

```suggestion
@click.argument('application-tags', type=click.File('r'), help='json formatted application tags')
```